paperless-ngx.json

```
@@ -0,0 +1,113 @@
+                    "source_container": "paperless-db"
+                }
+            ]
+        },
+        "containers": {
+            "paperless": {
+                "image": "ghcr.io/paperless-ngx/paperless-ngx",
+                "launch_order": 3,
+                "ports": {
+                    "8000": {
+                        "description": "Port for web interface. Suggested default: 8420.",
+                        "host_default": 8420,
+                        "label": "WebUI port",
+                        "protocol": "tcp",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/data": {
+                        "description": "Choose a Share for data file storage.",
+                        "label": "Config paperless Data - Storage"
+                    },
+                    "/media": {
+                        "description": "Choose a Share for media storage.",
+                        "label": "Config paperless Media - Storage"
+                    },
+                    "/export": {
+                        "description": "Choose a Share for exported documents.",
+                        "label": "Config paperless Export - Storage"
+                    },
+                    "/consume": {
+                        "description": "Choose a Share for importing documents.",
+                        "label": "Config paperless Import - Storage"
+                    }
+                },
+                "environment": {
+                    "PAPERLESS_REDIS": {
+                        "description": "Set to redis://paperless-broker:6379.Redis connection URL. Do not change unless you know what you are doing.",
+                        "value": "redis://paperless-broker:6379",
+                        "label": "Redis URL"
+                    },
+                    "PAPERLESS_DBHOST": {
+                        "description": "Set to paperless-db. Hostname of the PostgreSQL database container.",
+                        "value": "paperless-db",
+                        "label": "Database Host"
+                    },
+                    "PAPERLESS_TIME_ZONE": {
+                        "description": "Enter timezone",
+                        "value": "Europe/Berlin",
+                        "label": "Timezone"
+                    }
+                }
+            },
+            "paperless-broker": {
+                "image": "docker.io/library/redis",
+                "tag": "7",
+                "launch_order": 1,
+                "volumes": {
+                    "/data": {
+                        "description": "Choose a Share for Redis data.",
+                        "label": "Redis Data Share"
+                    }
+                }
+            },
+            "paperless-db": {
+                "image": "docker.io/library/postgres",
+                "tag": "16",
+                "launch_order": 2,
```

```suggestion
                "launch_order": 2,
                "opts": [
                    [
                        "-e",
                        "POSTGRES_DB=paperless"
                    ],
                    [
                        "-e",
                        "POSTGRES_USER=paperless"
                    ],
                    [
                        "-e",
                        "POSTGRES_PASSWORD=paperless"
                    ]
                ],
```

Since these values seem to be the default for the paperless container itself, unless we also want to explicitly expose those values in the paperless container, I would suggest to go with the default values, and hence being able to hide them from user entry.